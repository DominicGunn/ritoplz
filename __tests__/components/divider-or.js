'use strict'

/* eslint-env jest */

import renderer from 'react-test-renderer'

import DividerOr from '../../components/ui/divider-or'

describe('ui divider-or component', () => {
  describe('snapshots', () => {
    it('should render divider-or', () => {
      const tree = renderer.create(<DividerOr />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
