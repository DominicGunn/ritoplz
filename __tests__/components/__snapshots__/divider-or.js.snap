// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui divider-or component snapshots should render divider-or 1`] = `
<div
  data-jsx={4137905713}
>
  or
</div>
`;
